tools/make/kube.mk

```diff
@@ -45,3 +45,20 @@ kube-undeploy: kube-uninstall ## Uninstall the Envoy Gateway controller into the
 .PHONY: run-kube-local ## Run EG locally.
 run-kube-local: kube-install
 	hack/run-kube-local.sh
+
+.PHONY: gateway-conformance 
+gateway-conformance: create-conformance-cluster kube-deploy run-gateway-conformance delete-conformance-cluster ## Create a kind cluster, deploy EG into it, run Gateway API conformance, and clean up.
+
+.PHONY: create-conformance-cluster
+create-conformance-cluster: ## Create a kind cluster suitable for running Gateway API conformance.
+	hack/create-conformance-cluster.sh
+
+.PHONY: run-gateway-conformance
+run-gateway-conformance: ## Run Gateway API conformance.
+	kubectl wait --timeout=5m -n gateway-system deployment/gateway-api-admission-server --for=condition=Available
+	kubectl apply -f internal/provider/kubernetes/config/samples/gatewayclass.yaml
```

Comment:
doesnt this need to be applied to the `envoy-gateway-system` namespace ?

Reply:
It's cluster-scoped.